pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -320,7 +320,7 @@
     "hangouts" = ps: with ps; [ ]; # missing inputs: hangups
     "harman_kardon_avr" = ps: with ps; [ ]; # missing inputs: hkavr
     "harmony" = ps: with ps; [ aioharmony];
-    "hassio" = ps: with ps; [ aiohttp-cors sqlalchemy]; # missing inputs: home-assistant-frontend
```

Comment:
Please generate this file using the latest `update.py`. This version is missing transitive dependencies.

Reply:
This is in fact the latest version of parse-requirements.py:

```
commit d9068791d9be53a510fcd40f7379ace6fa4a12f7
Author: Jörg Thalheim <joerg@thalheim.io>
Date:   Sun Jun 21 16:07:30 2020 +0100

    home-assistant: also collect after_dependencies
```

Reply:
Resolved the dependency tree by hand:

- [hassio](https://github.com/home-assistant/core/blob/dev/homeassistant/components/hassio/manifest.json)
  - [http](https://github.com/home-assistant/core/blob/dev/homeassistant/components/http/manifest.json)
    - requires `aiohttp-cors`
  - [panel_custom](https://github.com/home-assistant/core/blob/dev/homeassistant/components/panel_custom/manifest.json)
    - [frontend](https://github.com/home-assistant/core/blob/dev/homeassistant/components/frontend/manifest.json)
      - [api](https://github.com/home-assistant/core/blob/dev/homeassistant/components/api/manifest.json)
        - http
      - [auth](https://github.com/home-assistant/core/blob/dev/homeassistant/components/auth/manifest.json)
        - http
      - [config](https://github.com/home-assistant/core/blob/dev/homeassistant/components/config/manifest.json)
        - http
      - [device_automation](https://github.com/home-assistant/core/blob/dev/homeassistant/components/device_automation/manifest.json)
      - http
      - [lovelace](https://github.com/home-assistant/core/blob/dev/homeassistant/components/lovelace/manifest.json)
      - [onboarding](https://github.com/home-assistant/core/blob/dev/homeassistant/components/onboarding/manifest.json)
        - auth
        - http
        - [person](https://github.com/home-assistant/core/blob/dev/homeassistant/components/person/manifest.json)
          - [device_tracker](https://github.com/home-assistant/core/blob/dev/homeassistant/components/device_tracker/manifest.json)
            - [zone](https://github.com/home-assistant/core/blob/dev/homeassistant/components/zone/manifest.json)
      - [search](https://github.com/home-assistant/core/blob/dev/homeassistant/components/search/manifest.json)
        - websocket_api
        - [scene](https://github.com/home-assistant/core/blob/dev/homeassistant/components/scene/manifest.json)
        - [group](https://github.com/home-assistant/core/blob/dev/homeassistant/components/group/manifest.json)
        - [automation](https://github.com/home-assistant/core/blob/dev/homeassistant/components/automation/manifest.json)
          - device_automation
          - [webhook](https://github.com/home-assistant/core/blob/dev/homeassistant/components/webhook/manifest.json)
            - http
        - [script](https://github.com/home-assistant/core/blob/dev/homeassistant/components/script/manifest.json)
      - [system_log](https://github.com/home-assistant/core/blob/dev/homeassistant/components/system_log/manifest.json)
        - http
      - [websocket_api](https://github.com/home-assistant/core/blob/dev/homeassistant/components/websocket_api/manifest.json)
        - http


The only requirement I can find in this tree is `aiohttp-cors`. In fact the only manifests that request `sqlalchemy` are that of the `recorder` and `sql` component.

Reply:
Fair enough. Thanks for clarification.